classes/dev/ComposerScript.php

```diff
@@ -32,4 +32,45 @@ public static function isoFileCheck(): void
             throw new Exception("The ISO639-2b file {$iso6392bFile} does not exist.");
         }
     }
+
+    /**
+     * A post-install-cmd custom composer script that
+     * creates languages.json from downloaded Weblate languages.csv.
+     */
+    public static function weblateFilesDownload(): void
+    {
+        try {
+            $dirPath = dirname(__FILE__, 3) . "/lib/weblateLanguages";
+            $urlCsvFilePath = "$dirPath/urlCsv.txt";
```

Comment:
I wasn't convinced that weblate wouldn't change the file name or location at some point in the future. Now one could just change the url in the file without updating ojs just for that.

Reply:
I think we can have the URL here, in a variable - it will be the same for the user ifhe/she needs to change it - to edit this or the other file. This way we do not need that extra file and that file reading...